### PR TITLE
Update excursions.html

### PR DESCRIPTION
**Pull request by mveyette**
Updated brewery tour info

### CODE REVIEW

File: pagesrc/excursions.html [hunk 1]
```diff
@@ -35,13 +35,13 @@ <h2><span>Excursions</span></h2>
...
-	      <li>Boston Beer Company (Samuel Adams) - Boston’s largest brewery:</li>
-<li>Mystic Brewery - Experts in traditional styles like saison, gruit, and wild ales.</li>
-	      <li>Night Shift Brewing - Brewers of Boston’s most unique beers like their horchata-style milk stout.</li>
-	      </ul>
-Price: 85$ (includes bus tranportation, beer tastings, taxes and fees)
+	      <li>Aeronaut Brewing Co. - Brewer of unique beers located in an art-centric community space</li>
+	      <li>Somerville Brewing Co. - A great local hangout known for their clever beer names</li>
+	      <li>Mystic Brewery - Experts in traditional styles like saison, gruit, and wild ales.</li>
+	    </ul>
+Price: 75$ (includes bus tranportation, beer tastings, and fees)
```
Comment (hamogu) [inline pagesrc/excursions.html]: I'm not sure I'm comfortable with decreasing the price to 75 $. How certain are you? We don't have the exact price for the bus. Also, if we need 75 $ and 30 people to pay for the bus and only 28 sign up, we'll have to cancel (which would be annoying and possibly loose deposits that we have already paid) or subsidize the tour from some other source (and particularly if the conference itself has fewer people sign up than we expect, we'll be short of money everywhere). On the other hand, if we collect too much money, we can always get a pretzel with mustard for everyone and a glass with a logo. 
I understand that 85 $ is high and I'd like to get that down, but not to the level where we are in risk of loosing money on this excursion. Does that convince you or do you insist on 75 $?

File: pagesrc/excursions.html [hunk 2]
```diff
@@ -35,13 +35,13 @@ <h2><span>Excursions</span></h2>
         <div class="c4 about ta-c">
           <img src="images/excur-beer.jpg" alt="" />
           <p class="name">Boston Craft Brewing Tour</p>
-          <p class="position"> Take a grand tour of American craft brewing! Learn about the history and process of craft brewing from a Boston beer expert as you tour and taste your way around three local breweries:
+          <p class="position"> Take a grand tour of American craft brewing! Learn about the history and process of craft brewing from local brewers as you tour and taste your way around three Boston-area breweries:
 	    <ul>
-	      <li>Boston Beer Company (Samuel Adams) - Boston’s largest brewery:</li>
-<li>Mystic Brewery - Experts in traditional styles like saison, gruit, and wild ales.</li>
-	      <li>Night Shift Brewing - Brewers of Boston’s most unique beers like their horchata-style milk stout.</li>
```
Comment (hamogu) [inline pagesrc/excursions.html:42]: Just and idea: Should we add links to those breweries? We did not have that before, but now that I look for it, it seems like a reasonable idea... What do you think?

Comment (mveyette) [reply pagesrc/excursions.html:42]: Sure. I will add those and resubmit the pull.

File: pagesrc/excursions.html (hunk 1 above)
Comment (mveyette) [reply pagesrc/excursions.html]: We can leave it at $85 on the website for now and wait until Jenine solidifies plans (esp with the bus) to reassess before registration opens.

**Comment by hamogu** [comment]
tested and updated

**Pull request merged**